pkg/volume/csi/csi_plugin.go

```diff
@@ -59,9 +62,54 @@ func ProbeVolumePlugins() []volume.VolumePlugin {
 // volume.VolumePlugin methods
 var _ volume.VolumePlugin = &csiPlugin{}
 
+type csiDriver struct {
+	driverName     string
+	driverEndpoint string
+}
+
+type csiDriversStore struct {
+	driversMap map[string]csiDriver
```

Comment:
Curious why you are no longer using the `sync.Map`  type any more.

Reply:
There was a comment about sync.Map suggesting that it is not strong typed. So I switched back to a simple map with mutex.